Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Program.cs

```diff
@@ -19,7 +19,16 @@ static void Main(string[] args)
     }
 }
 #endif
-#if NetCore
+#if NET6_0_OR_GREATER
```

Comment:
Do we need this level of granularity? Or can we just keep the original `#if NetCore` conditional? 
([the ext package targets net6](https://github.com/Azure/azure-functions-dotnet-worker/blob/e4f4b4db4902454b35ae18a4f9948b896f7941f6/extensions/Worker.Extensions.Http.AspNetCore/src/Worker.Extensions.Http.AspNetCore.csproj#L10))